client/src/pages/private/SiteParticipantsTable.js

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import _orderBy from 'lodash/orderBy';
+import Grid from '@material-ui/core/Grid';
+import { Box } from '@material-ui/core';
+import store from 'store';
+import { Table } from '../../components/generic';
+
+const columns = [
+  { id: 'pptId', name: 'ID' },
+  { id: 'pptName', name: 'Name' },
+  { id: 'status', name: 'Status' },
+  { id: 'emailAddress', name: 'Email Address' },
+  { id: 'phoneNumber', name: 'Phone Number' },
+];
+
+export default ({ siteId }) => {
+  const [order, setOrder] = useState('asc');
+  const [isLoadingData, setLoadingData] = useState(false);
+  const [isPendingRequests, setIsPendingRequests] = useState(true);
+  const [rows, setRows] = useState([]);
+  const [fetchedRows, setFetchedRows] = useState([]);
+
+  const [orderBy, setOrderBy] = useState(columns[4].id);
+
+  const handleRequestSort = (event, property) => {
+    const isAsc = orderBy === property && order === 'asc';
+    setOrder(isAsc ? 'desc' : 'asc');
+    setOrderBy(property);
+  };
+
+  const sort = (array) => _orderBy(array, [orderBy, 'operatorName'], [order]);
+
+  useEffect(() => {
+    const fetchParticipants = async () => {
+      setLoadingData(true);
+      const response = await fetch(`/api/v1/employer-sites/${siteId}/participants`, {
```

Comment:
Previously, @fwkendall mentioned this would be better suited to come with the request for site details.

Reply:
I mistook what was being said. Good as is.